### ISSUE TITLE
When I try to run Evolv.io, this happens

### ISSUE THREAD

**Issue by Thecoder3281F** [evolvio/evolv.io#196]
`Exception in thread "main" java.lang.ExceptionInInitializerError
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at processing.core.PApplet.<clinit>(PApplet.java:119)`

**Comment by UnknownEvil** [comment]
Only thing I can come up with is wrong version of Java. Otherwise, any more logs?

**Comment by Thecoder3281F** [comment]
I think the array value is wrong.

**Comment by UnknownEvil** [comment]
@Thecoder3281F
Yes, the array is broken, but it's in PApplet. Which we do not have control over in this Repo
